modules/mongodb/manifests/server.pp

```diff
@@ -28,6 +28,7 @@
   $oplog_size = undef,
   $replicaset_members = {},
   $development = false,
+  $s3_backups = false,
```

Comment:
Yes, server.pp is the entry point for backup.pp so I thought it would also be a better place to include s3backup/* classes. I want to be able to include all the s3backup classes if this is turned on, regardless of whether regular backups are turned on.